src/components/CompactCard.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { connect } from "react-redux";
+import moment from "moment";
+import Checkbox from "./Checkbox";
+import { updateStreak } from "../actions";
+import "../css/main.css";
+
+class CompactCard extends React.Component {
+  renderStreak(streak, endDate) {
```

Comment:
This looks like it might duplicate some code from the larger habit card component. Maybe this could be pulled out into a helper function?

Reply:
Instead I decided to just merge the compact card and expanded card, to avoid some other duplication in the file.